target/linux/ath79/dts/ar9344_devolo_dlan2-2400-ac.dts

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,dlan2-2400-ac", "devolo,magic2wifi", "qca,ar9344";
+
+	aliases {
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan {
+			label = "devolo:status:wlan";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
```

Comment:
What about phy0tpt?

Reply:
There are only 3 leds, and two are together. I think it´s more important to use the second one for the uplink, and there are more devices using 2.4G than 5G...

Reply:
Okay, so the vendor indicated both 2.4 GHz and 5 GHz with this LED?
Maybe someone else knows whether we already have an example case for that.

Note that the policy in OpenWrt is to keep the LEDs as the vendor uses them. So, even if you wanted to repurpose another LED for separating WiFi indication, it wouldn't be accepted anyway ...

Reply:
The vendor keeps the wifi led on or off all the time, user chooses the behaviour, and the same goes for the PLC led, but with PLC led vendor switch it to red if you lose the PLC uplink.
Since PLC uplink can´t be monitored yet, the closest thing to identify your uplink is watching for traffic on port 2, but it didn´t work when I tried, so I am doing it at the eth0...